tests/DependencyInjection/SonataAdminExtensionTest.php

```diff
@@ -65,208 +84,81 @@ public function testHasSecurityRoleParameters()
 
     public function testAssetsCanBeAddedAndRemoved()
     {
         $this->container->setParameter('kernel.bundles', []);
+        $extraStylesheets = ['foo/bar.css', 'bar/quux.css'];
+        $extraJavascripts = ['foo/bar.js', 'bar/quux.js'];
+        $removeStylesheets = ['bundles/sonataadmin/vendor/admin-lte/dist/css/skins/skin-black.min.css', 'bundles/sonataadmin/vendor/jqueryui/themes/base/jquery-ui.css'];
+        $removeJavascripts = ['bundles/sonataadmin/vendor/readmore-js/readmore.min.js', 'bundles/sonataadmin/jquery/jquery.confirmExit.js'];
         $this->load([
             'assets' => [
-                'extra_stylesheets' => [
-                    'foo/bar.css',
-                    'bar/quux.css',
-                ],
-                'remove_stylesheets' => [
-                    'bundles/sonataadmin/vendor/admin-lte/dist/css/skins/skin-black.min.css',
-                    'bundles/sonataadmin/vendor/jqueryui/themes/base/jquery-ui.css',
-                ],
-                'extra_javascripts' => [
-                    'foo/bar.js',
-                    'bar/quux.js',
-                ],
-                'remove_javascripts' => [
-                    'bundles/sonataadmin/vendor/readmore-js/readmore.min.js',
-                    'bundles/sonataadmin/jquery/jquery.confirmExit.js',
-                ],
+                'extra_stylesheets' => $extraStylesheets,
+                'remove_stylesheets' => $removeStylesheets,
+                'extra_javascripts' => $extraJavascripts,
+                'remove_javascripts' => $removeJavascripts,
             ],
         ]);
         $stylesheets = $this->container->getDefinition('sonata.admin.pool')->getArgument(3)['stylesheets'];
-        $javascripts = $this->container->getDefinition('sonata.admin.pool')->getArgument(3)['javascripts'];
 
-        $this->assertEquals($stylesheets, [
-            'bundles/sonatacore/vendor/bootstrap/dist/css/bootstrap.min.css',
-            'bundles/sonatacore/vendor/components-font-awesome/css/font-awesome.min.css',
-            'bundles/sonatacore/vendor/ionicons/css/ionicons.min.css',
-            'bundles/sonataadmin/vendor/admin-lte/dist/css/AdminLTE.min.css',
-            'bundles/sonataadmin/vendor/iCheck/skins/square/blue.css',
-            'bundles/sonatacore/vendor/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css',
-            'bundles/sonatacore/vendor/select2/select2.css',
-            'bundles/sonatacore/vendor/select2-bootstrap-css/select2-bootstrap.min.css',
-            'bundles/sonataadmin/vendor/x-editable/dist/bootstrap3-editable/css/bootstrap-editable.css',
-            'bundles/sonataadmin/css/styles.css',
-            'bundles/sonataadmin/css/layout.css',
-            'bundles/sonataadmin/css/tree.css',
-            'foo/bar.css',
-            'bar/quux.css',
-        ]);
+        $this->assertSame(array_merge(array_diff($this->defaultStylesheets, $removeStylesheets), $extraStylesheets), $stylesheets);
```

Comment:
It seems that his line is over 120 characters, could you please break it? Thank you